cassandra/cluster.py

```diff
@@ -2073,6 +2097,24 @@ def _handle_results(success, result):
             log.debug("[control connection] Fetched user type info for %s.%s, rebuilding metadata", keyspace, usertype)
             types_result = dict_factory(*types_result.results) if types_result.results else {}
             self._cluster.metadata.usertype_changed(keyspace, usertype, types_result)
+        elif function:
+            # user defined function within this keyspace changed
+            where_clause = " WHERE keyspace_name = '%s' AND function_name = '%s' AND signature = [%s]" \
+                           % (keyspace, function.name, ','.join("'%s'" % t for t in function.type_signature))
```

Comment:
Would it be a useful feature to allow refreshing by function/aggregate "name" alone? if the "type_signature" is None it would refresh all functions/aggregates of the same name? Maybe this would collide with a function with no parameters. I say this because building the type signature could be difficult and providing the name allow would be more user friendly.